src/community/jms-cluster/jms-geoserver/src/main/java/org/geoserver/cluster/impl/handlers/configuration/JMSLoggingHandler.java

```diff
@@ -37,7 +36,7 @@ protected void omitFields(final XStream xstream) {
     @Override
     public boolean synchronize(LoggingInfo info) throws Exception {
         if (info == null) {
-            throw new NullArgumentException("Incoming object is null");
+            throw new NullPointerException("Incoming object is null");
```

Comment:
this appears to be the only substantial difference right? Use of NullPointerException ...

Reply:
Yes and no. Yes in that lang3 removes NullArgumentException, but no in that there are other changes (e.g. string escapes, signature changes). All the changes are pretty small, just a lot of them.